### ISSUE TITLE
how to decode the prediction？

### ISSUE THREAD

**Issue by Asunatan** [WeixiongLin/PMC-CLIP#20]
It seems that the author did not provide the decode code. Can I use the decode code in [OpenCLIP](https://github.com/mlfoundations/open_clip) to decode the text?

**Comment by coder1165** [comment]
Did you solve it?I also have the same problem.

**Comment by WeixiongLin** [comment]
Yes, sure you can add your decoding code. However, PMC-CLIP is implemented without decoding module, so you may need to further finetune after you implement your own model.

**Comment by WeixiongLin** [comment]
I am closing this issue now, feel free to reopen it if you need further help.
